src/main/kotlin/com/hedvig/underwriter/serviceIntegration/productPricing/dtos/contract/NorwegianTravelLineOfBusiness.kt

```diff
@@ -3,4 +3,11 @@ package com.hedvig.underwriter.serviceIntegration.productPricing.dtos.contract
 enum class NorwegianTravelLineOfBusiness {
     REGULAR,
     YOUTH;
+
+    companion object {
+        fun from(isYouth: Boolean): NorwegianTravelLineOfBusiness = when (isYouth) {
+            true -> YOUTH
+            false -> REGULAR
+        }
```

Comment:
```suggestion
        fun from(isYouth: Boolean): NorwegianTravelLineOfBusiness = if (isYouth) YOUTH else REGULAR
```
I think this is cleaner but do as you please :)